quantum/keymap_extras/keymap_dvorak_fr.h

```diff
@@ -0,0 +1,131 @@
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/* Dvorak for the French language
+ * Version: 2
+ *
+ * The layout is designed by Francis Leboutte <dvorak-fr@algo.be>
+ *
+ * Source: https://algo.be/ergo/dvorak-fr.html
+ */
+
+#pragma once
+
+#include "keymap.h"
+
+// clang-format off
+
+/*
+ * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───────┐
+ * │ « │ » │ / │ - │ è │ \ │ ^ │ ( │ ` │ ) │ _ │ [ │ ] │       │
+ * ├───┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─────┤
+ * │     │ : │ ' │ é │ G │ . │ H │ V │ C │ M │ K │ Z │ ¨ │     │
+ * ├─────┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┬──┴┐    │
+ * │      │ O │ A │ U │ E │ B │ F │ S │ T │ N │ D │ W │ ~ │    │
+ * ├────┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴─┬─┴───┴────┤
+ * │    │ à │ ; │ Q │ , │ I │ Y │ X │ R │ L │ P │ J │          │
+ * └────┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴───┴──────────┘
+ */
+// Row 1
+#define DF_LDAQ KC_GRV  // «
+#define DF_RDAQ KC_1    // »
+#define DF_SLSH KC_2    // /
+#define DF_MINS KC_3    // -
+#define DF_EGRV KC_4    // è
+#define DF_BSLS KC_5    // (backslash)
+#define DF_CIRC KC_6    // ^ (dead)
+#define DF_LPRN KC_7    // (
+#define DF_GRV  KC_8    // ` (dead)
+#define DF_RPRN KC_9    // )
+#define DF_UNDS KC_0    // _
+#define DF_LBRC KC_MINS // [
+#define DF_RBRC KC_EQL  // ] (dead)
+// Row 2
+#define DF_COLN KC_Q    // :
+#define DF_QUOT KC_W    // '
+#define DF_EACU KC_E    // é
+#define DF_G    KC_R    // G
+#define DF_DOT  KC_T    // .
+#define DF_H    KC_Y    // H
+#define DF_V    KC_U    // V
+#define DF_C    KC_I    // C
+#define DF_M    KC_O    // M
+#define DF_K    KC_P    // K
+#define DF_Z    KC_LBRC // Z
+#define DF_DIAE KC_RBRC // ¨ (dead)
+// Row 3
+#define DF_O    KC_A    // O
+#define DF_A    KC_S    // A
+#define DF_U    KC_D    // U
+#define DF_E    KC_F    // E
+#define DF_B    KC_G    // B
+#define DF_F    KC_H    // F
+#define DF_S    KC_J    // S
+#define DF_T    KC_K    // T
+#define DF_N    KC_L    // N
+#define DF_D    KC_SCLN // D
+#define DF_W    KC_QUOT // W
+#define DF_TILD KC_NUHS // ~ (dead)
+// Row 4
+#define DF_AGRV KC_NUBS // à
+#define DF_SCLN KC_Z    // ;
+#define DF_Q    KC_X    // Q
+#define DF_COMM KC_C    // ,
+#define DF_I    KC_V    // I
+#define DF_Y    KC_B    // Y
+#define DF_X    KC_N    // X
+#define DF_R    KC_M    // R
+#define DF_L    KC_COMM // L
+#define DF_P    KC_DOT  // P
+#define DF_J    KC_SLSH // J
```

Comment:
```suggestion
#define DV_LDAQ KC_GRV  // «
#define DV_RDAQ KC_1    // »
#define DV_SLSH KC_2    // /
#define DV_MINS KC_3    // -
#define DV_EGRV KC_4    // è
#define DV_BSLS KC_5    // (backslash)
#define DV_CIRC KC_6    // ^ (dead)
#define DV_LPRN KC_7    // (
#define DV_GRV  KC_8    // ` (dead)
#define DV_RPRN KC_9    // )
#define DV_UNDS KC_0    // _
#define DV_LBRC KC_MINS // [
#define DV_RBRC KC_EQL  // ]
// Row 2
#define DV_COLN KC_Q    // :
#define DV_QUOT KC_W    // '
#define DV_EACU KC_E    // é
#define DV_G    KC_R    // G
#define DV_DOT  KC_T    // .
#define DV_H    KC_Y    // H
#define DV_V    KC_U    // V
#define DV_C    KC_I    // C
#define DV_M    KC_O    // M
#define DV_K    KC_P    // K
#define DV_Z    KC_LBRC // Z
#define DV_DIAE KC_RBRC // ¨ (dead)
// Row 3
#define DV_O    KC_A    // O
#define DV_A    KC_S    // A
#define DV_U    KC_D    // U
#define DV_E    KC_F    // E
#define DV_B    KC_G    // B
#define DV_F    KC_H    // F
#define DV_S    KC_J    // S
#define DV_T    KC_K    // T
#define DV_N    KC_L    // N
#define DV_D    KC_SCLN // D
#define DV_W    KC_QUOT // W
#define DV_TILD KC_NUHS // ~ (dead)
// Row 4
#define DV_AGRV KC_NUBS // à
#define DV_SCLN KC_Z    // ;
#define DV_Q    KC_X    // Q
#define DV_COMM KC_C    // ,
#define DV_I    KC_V    // I
#define DV_Y    KC_B    // Y
#define DV_X    KC_N    // X
#define DV_R    KC_M    // R
#define DV_L    KC_COMM // L
#define DV_P    KC_DOT  // P
#define DV_J    KC_SLSH // J
```

Reply:
I'm not sure about this changes. It will conflict with dvorak if someone wants to use both layouts on a board